GitUI/Plugin/FailedPluginWrapper.cs

```diff
@@ -7,7 +7,7 @@ namespace GitUI;
 
 internal partial class FailedPluginWrapper : IGitPlugin
 {
-    [GeneratedRegex("\\\"GitExtensions.([^\"]+)\\\"")]
+    [GeneratedRegex(@"""GitExtensions.([^""]+)""", RegexOptions.ExplicitCapture)]
```

Comment:
ditto

Reply:
All will be handled the same, except that brackets are needed here

Reply:
```suggestion
    [GeneratedRegex(@"[""]GitExtensions\.([^""]+)[""]", RegexOptions.ExplicitCapture)]
```
might be clearer - optional

Reply:
In some way keeping the string minimal is preferred, I am thinking that I should have changed that.
But the double quote needed i slightly clearer here, so I will not make changes unless suggested